api/turmas/tui.py

```diff
@@ -26,13 +26,19 @@ def detail_turma(turma, title="Detalhes da Turma:"):
     fake_client_name = turma["fake_client"]["name"]
     fake_client_email = turma["fake_client"]["email"]
     students_count = len(turma["students"])
+    students = turma["students"]
+    count = 0
 
     print(title)
     print(f"Id: {id}")
     print(f"Nome: {name}")
     print(f"Líder de Grupo: {group_leader_name} <{group_leader_email}>")
     print(f"Fake Client: {fake_client_name} <{fake_client_email}>")
     print(f"Total de Alunos: {students_count}")
+    while count < len(students):
```

Comment:
Acredito que poderia ser um `for student in students: `, assim não precisaria de um `count`